server/src/test/scala/services/arrivals/PcpPaxSpec.scala

```diff
@@ -6,10 +6,30 @@ import org.specs2.mutable.Specification
 
 class PcpPaxSpec extends Specification {
```

Comment:
I think we may have a missing case here:
This will only use API when we have no other source of pax nos. What about where we have ACL pax nos, no live pax nos and API - do we not want to use API in this case?

Reply:
Actually maybe this is what was planned. I don't know!